canonical-web-and-design/snapcraft.io
# Remove sqlite cached session

## Done

- Remove sqlite cached session

## Issue / Card

Fixes https://github.com/canonical-web-and-design/snap-squad/issues/1655

## QA

- Pull the branch
- Run the site using the dotrun snap with `$ dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8004/
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Check site
